public_html/wp-content/plugins/wordcamp-remote-css/platforms/github.php

```diff
@@ -151,10 +148,98 @@ function should_authenticate_url( $request_url_parts, $request_args ) {
  * @return string
  */
 function decode_api_response( $response_body, $remote_css_url ) {
-	if ( false !== strpos( $remote_css_url, GITHUB_API_HOSTNAME ) ) {
+	if ( false !== strpos( $remote_css_url, API_HOSTNAME ) ) {
 		$response_body = json_decode( $response_body );
 		$response_body = base64_decode( $response_body->content );
 	}
 
 	return $response_body;
 }
+
+/**
+ * Fetch an Authorization token for an authenticated Github API request.
+ */
+function get_authorization_header() {
+	$token = get_app_install_token();
+	// Upon failure, just return an empty header, as GitHub will accept that at the lower rate limit temporarily.
```

Comment:
That's a good idea 👍

Reply:
Just noting for posterity, I've since double checked that sending an empty `Authorization:` header as done here does currently work and falls back to the regular 60 limit.